### PR TITLE
feature: add openjourney models

### PR DESCRIPTION
**Pull request by zmactep**
This commit makes possible to use OpenJourney models (v1 and v2).

### CODE REVIEW

**Review by Titaniumtown — APPROVED** [review]
LGTM, works perfectly

File: imaginairy/config.py [hunk 1]
```diff
@@ -97,6 +97,20 @@ class ModelConfig:
         weights_url="https://huggingface.co/imaginairy/instruct-pix2pix/resolve/ea0009b3d0d4888f410a40bd06d69516d0b5a577/instruct-pix2pix-00-22000-pruned.ckpt",
         default_image_size=512,
         default_negative_prompt="",
+    ),
+    ModelConfig(
+        short_name="openjourney",
+        config_path="configs/stable-diffusion-v1.yaml",
+        weights_url="https://huggingface.co/prompthero/openjourney/resolve/main/mdjrny-v4.ckpt",
```
Comment (brycedrennan) [inline imaginairy/config.py:104]: I want to pin this to specific commits so we don't have future issues.

**Review by brycedrennan — COMMENTED** [review]
Great idea. I'm considering adding functionality such that the necessary token "mdjrny-v4 style" is automatically added in.  Maybe a prompt template feature with default prompts per model.

**Comment by brycedrennan** [comment]
I pinned the hashes and merged this in. thank you!

**Pull request closed without merging**